go.mod

```diff
@@ -1,6 +1,6 @@
 module tailscale.com
 
-go 1.23
+go 1.23.0
```

Comment:
Did we end up with this change accidentally?

Reply:
Oops yeah, this was me being lazy about fixing my VSCode environment 😅 reverted.